ast/builtins.go

```diff
@@ -2289,6 +2290,17 @@ var CryptoX509ParseCertificateRequest = &Builtin{
 	),
 }
 
+var CryptoX509ParseKeyPair = &Builtin{
+	Name:        "crypto.x509.parse_x509_keypair",
+	Description: "Returns a valid key pair",
+	Decl: types.NewFunction(
+		types.Args(
+			types.Named("cert", types.S).Description("PEMencoded x509 certificate string"),
```

Comment:
question: based on the implementation, it seems like this also accepts base64 encoded DER data too?

In `parse_and_verify_certificates` we phrase that behaviour like this: 'from the given string containing PEM
or base64 encoded DER certificates'